dgp/lib/trajectory_ingestor.py

```diff
@@ -60,8 +60,14 @@ def import_trajectory(filepath, delim_whitespace=False, interval=0,
         Pandas DataFrame of ingested Trajectory data.
 
     """
-
-    df = pd.read_csv(filepath, delim_whitespace=delim_whitespace, header=None,
+    if filepath[-12:-4] == 'TerraPOS':
+        print('You gave me a TerraPOS trajectory.')
+        df = pd.read_csv(filepath, delim_whitespace=delim_whitespace, header=None,
+                     engine='python', na_filter=False, skiprows=skiprows, sep='\s+')
+        # print(df.describe())
+    else:
+        print('You gave me a Waypoint trajectory.')
+        df = pd.read_csv(filepath, delim_whitespace=delim_whitespace, header=None,
```

Comment:
I think it would be better if we just exposed `sep` and `engine` as arguments in `import_trajectory`. Checking for a string in the filename is brittle.